examples/example_seismic.py

```diff
@@ -41,104 +41,78 @@
-    #format p = np.arange (starting pressure, ending pressure, pressure step) (in Pa)
-    p = np.arange(1.0e9,360.0e9,5e9)
-    depths = np.array([s.depth(pr) for pr in p])
-    #we could also just specify some depth levels directly like this:
-    #depths = np.arange(35e3,5600e3,100e3)
-    #we could also use the data points where the seismic model is specified:
-    depths = s.internal_depth_list()
-
-    #now evaluate everything at the given depths levels (using interpolation)
-    pressures, density, v_p, v_s, v_phi = s.evaluate_all_at(depths)
+    #List of seismic 1D models 
+    models=[burnman.seismic.PREM(), burnman.seismic.STW105(),burnman.seismic.AK135(),burnman.seismic.IASP91()]
+    colors = ['r','b','m','k']
+    # Variables to plot
+    vars=['pressure','gravity','v_p','v_s','v_phi','density']#,
+    units=['Pa','m/s^2','m/s','m/s','m/s','kg/m^3','Pa','m/s^2']
+    plt.figure(figsize=(14,8))
     
-    # plot vs and vp and v_phi (note that v_phi is computed!)
-    plt.subplot(2,2,1)
-    plt.title('prem')
-    plt.plot(depths/1.e3,v_p/1.e3,'+-r', label='v_p')
-    plt.plot(depths/1.e3,v_s/1.e3,'+-b', label='v_s')
-    plt.plot(depths/1.e3,v_phi/1.e3,'--g', label='v_phi')
-    plt.legend(loc='lower left')
-    plt.xlabel('depth in km')
-    plt.ylabel('km/s')
-
-    # plot pressure,density vs depth from prem:
-    plt.subplot(2,2,2)
-    plt.title('prem')
-    plt.plot(depths/1.e3,pressures/1.e9,'-r', label='pressure')
-    plt.ylabel('GPa')
-    plt.xlabel('depth in km')
-    plt.legend(loc='upper left')
-    plt.twinx()
-    plt.ylabel('g/cc')
-    plt.plot(depths/1.e3,density/1.e3,'-b', label='density')
-    plt.legend(loc='lower right')
-
-
-    #now load fast and slow regionalized models (Lekic et al. 2012):
-    sslow = burnman.seismic.Slow()
-    depths2 = sslow.internal_depth_list()
-    pressures2, density2, v_p2, v_s2, v_phi2 = sslow.evaluate_all_at(depths2)
-
-    sfast = burnman.seismic.Fast()
-    depths3 = sfast.internal_depth_list()
-    pressures3, density3, v_p3, v_s3, v_phi3 = sfast.evaluate_all_at(depths3)
 
+    # Run through models and variables
+    for a in range(len(vars)):
+        for m in range(len(models)):
+
+                # specify where we want to evaluate, here we map from pressure to depth
+                #1. format p = np.arange (starting pressure, ending pressure, pressure step) (in Pa)
+                #p = np.arange(1.0e9,360.0e9,1.e9)
+                #depths = np.array([models[m].depth(pr) for pr in p])
+                #2. we could also just specify some depth levels directly like this
+                #depths = np.arange(700e3,2800e3,100e3)
+                #3. we could also use the data points where the seismic model is specified over a depth range, this will bring out any discontinuities
+                #this is the preferred way to plot seismic discontinuities correctly
+                depths = models[m].internal_depth_list(mindepth=-1.e3, maxdepth=6372.1e3)
```

Comment:
Why are we starting from a negative minimum depth? Perhaps a comment about that?